Sources/MapboxDirections/Extensions/GeoJSON.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import CoreLocation
+
+struct Geometry: Codable {
```

Comment:
This file’s path implies that it contains extensions to Turf’s GeoJSON types, whereas in fact it defines a new private Geometry type. Move this file somewhere else and call it Geometry.swift.